BothamUI/BothamEmptyCaseViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  BothamEmptyCaseViewController.swift
+//  BothamUI
+//
+//  Created by Pedro Vicente Gomez on 13/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+public protocol BothamEmptyCaseViewController : BothamEmptyCaseUI {
```

Comment:
Or maybe expect table view instead of a view, I want to review how https://github.com/dzenbot/DZNEmptyDataSet is implemented first.